HousingRepairsOnlineApi.Tests/HelpersTests/SoREngineTests.cs

```diff
@@ -10,48 +12,29 @@ public class SoREngineTests
 
         public SoREngineTests()
         {
-            IDictionary<string, IDictionary<string, IDictionary<string, string>>> mapping = new Dictionary<string, IDictionary<string, IDictionary<string, string>>>()
-            {
-                {
-                    "kitchen",
-                    new Dictionary<string, IDictionary<string, string>>
-                    {
-                        {
-                            "cupboards", new Dictionary<string, string>
-                            {
-                                { "doorHangingOff", "N373049" },
-                                { "doorMissing", "N373049" },
-                            }
-                        },
-                        {
-                            "worktop", new Dictionary<string, string>
-                            {
-                                { "damaged", "N372005" },
-                            }
-                        },
-                    }
+            var json = @"{
+              ""kitchen"": {
+                ""cupboards"":{
+                  ""doorHangingOff"":""N373049"",
+                  ""doorMissing"":""N373049""
                 },
-                {
-                    "bathroom",
-                    new Dictionary<string, IDictionary<string, string>>
-                    {
-                        {
-                            "bath", new Dictionary<string, string>
-                            {
-                                {"bathTaps", "N631301"},
-
-                            }
-                        },
-                    }
+                ""worktop"": ""N372005""
+              },
+              ""bathroom"": {
+                ""bath"": {
+                  ""bathTaps"": ""N631301""
                 }
-            };
+              }
+            }";
+            var mapping = JsonConvert.DeserializeObject<IDictionary<string, dynamic>>(json);
```

Comment:
Using `JsonConvert` makes this test dependent upon on other code.
To ensure the test is only testing the `SorEngine` constructor, use a datastructure similar to the previous one (above).